src/System.Net.Requests/tests/HttpWebRequestTest.cs

```diff
@@ -19,14 +19,6 @@ public partial class HttpWebRequestTest
         private const string RequestBody = "This is data to POST.";
         private readonly byte[] _requestBodyBytes = Encoding.UTF8.GetBytes(RequestBody);
         private readonly NetworkCredential _explicitCredential = new NetworkCredential("user", "password", "domain");
-
-        private HttpWebRequest _savedHttpWebRequest = null;
-        private WebHeaderCollection _savedResponseHeaders = null;
-        private Exception _savedRequestStreamException = null;
-        private Exception _savedResponseException = null;
-
-        private int _requestStreamCallbackCallCount = 0;
-        private int _responseCallbackCallCount = 0;
```

Comment:
This is a good change, in that it's better to have this as local rather than instance state.  I'm curious though why it was necessary... tests on the same instance are never run in parallel by xunit, with it creating a new instance per test method invocation.